### PR TITLE
use the json accept header for talk api

### PR DESCRIPTION
**Pull request by camallen**
get the json rep of the resources, avoid the html redirect on the main repo

### CODE REVIEW

File: lib/talk_api_client.rb [hunk 1]
```diff
@@ -78,14 +78,14 @@ def create_connection(adapter)
   end
 
   def initial_request
-    request('get', '').body.each do |_, resource|
+    request('get', '', accept_header: "application/json").body.each do |_, resource|
```
Comment (edpaget) [inline lib/talk_api_client.rb]: Why is this one different?

Comment (parrish) [inline lib/talk_api_client.rb]: This won't need to be different after zooniverse/Talk-Api@c46b359160e435aa5 is deployed

Comment (camallen) [inline lib/talk_api_client.rb]: Well that makes it easier!

**Comment by edpaget** [comment]
We still need the other change though right @camallen?

**Comment by camallen** [comment]
@edpaget which other change?

**Comment by edpaget** [comment]
Nevermind. I'm having trouble reading today.

**Pull request closed without merging**